circle.yml

```diff
@@ -61,10 +61,10 @@ commands:
       browser:
         default: "electron"
         description: browser shortname to target
-        type: enum
-        enum: [ "chrome", "electron" ]
+        type: string
```

Comment:
I changed the `browser` to enum because of more browsers are coming. :smile: Either way is fine.

Please support more browsers in the near future! :bowing_woman:

Reply:
#1359 is super close :fox_face: